### DISCUSSION TITLE
Formatting the count matrix to 10x feature barcode matrix HDF5

### DISCUSSION THREAD

**Discussion by shregeno8** [campbio/celda#380] [Q&A] [answered]
Hi I used DecontX for generating the corrected count matrix after filtering for ambient RNA contamination. I was wondering if there is a way to convert the count matrix to 10x feature barcode matrix HDF5 format. 

Thanks in advance.

Regards
She

**Comment by joshua-d-campbell** [comment, marked as answer]
Hi @shregeno8, we have another package called [singleCellTK](https://www.bioconductor.org/packages/release/bioc/html/singleCellTK.html) which can export the data to AnnData format which is based on HDF5. DecontX can return a SingleCellExperiment object with the decontaminated counts. There is another Bioconductor package called [rhdf5](https://bioconductor.org/packages/release/bioc/html/rhdf5.html) which you can use to convert matrices to HDF5 objects as well.

**Reply by shregeno8** [reply]
Hi Josh,
Thank you very much. I will check them out.
Much Appreciated.

Regards
Shre
